addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -3906,19 +3906,6 @@ const SnippetOptionWidget = Widget.extend({
      * @returns {Promise<boolean>|boolean}
      */
     _computeWidgetVisibility: async function (widgetName, params) {
-        const moveUpOrLeft = widgetName === 'move_up_opt' || widgetName === 'move_left_opt';
-        const moveDownOrRight = widgetName === 'move_down_opt' || widgetName === 'move_right_opt';
-
-        if (moveUpOrLeft || moveDownOrRight) {
-            // The arrows are not displayed if the target is in a grid and if
-            // not in mobile view.
-            const isMobileView = weUtils.isMobileView(this.$target[0]);
-            if (this.$target[0].classList.contains('o_grid_item') && !isMobileView) {
```

Comment:
This move does not work, grid items arrow are now always visible :grimacing: And click on them crashes...

Reply:
Done!